tests/test_portability.py

```diff
@@ -9,22 +9,33 @@
 # set other CPU types for other architectures; Westmere supports cpuid but not avx2
 MIN_CPUS["x86_64"] = "Westmere"
 
+def check_ubuntu():
+   try:
+      with open("/etc/os-release") as f:
+         for line in f.readlines():
+            if "Ubuntu" in line:
+               return True
+   # if file /etc/os-release doesn't exist, we're not on Linux:
```

Comment:
It seems like `os-release` is for Linux distributions that use `systemd` and can be (has been) adopted by other distributions. This would work to detect Ubuntu, but perhaps the comment is a bit inaccurate?

Reply:
ACK